pallets/parachain-staking/src/inflation.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+//! Inspired by:
+//! - Moonbeam `pallet_parachain_staking`
+//! implementations.
+
+//! Helper methods for computing issuance based on inflation
+use crate::pallet::{BalanceOf, Config, Pallet};
+use frame_support::traits::Currency;
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::{PerThing, Perbill, RuntimeDebug};
+use substrate_fixed::{
+	transcendental::pow as floatpow,
+	types::{I32F32, I64F64},
+};
+
+const SECONDS_PER_YEAR: u32 = 31557600;
+const SECONDS_PER_BLOCK: u32 = 12;
+pub const BLOCKS_PER_YEAR: u32 = SECONDS_PER_YEAR / SECONDS_PER_BLOCK;
```

Comment:
we could add a YEARS const in `primitives/src/lib.rs` and use it from there

Reply:
Would you prefer 365.25 days per year or 365?

Reply:
365.25 now